azavea/raster-foundry
# New project page + navbar

## Overview

Redesigns the Projects page and navbar to match our new workflow plans.

### Checklist


### Demo
![rfnewinterface](https://cloud.githubusercontent.com/assets/447977/23809650/86960d6e-059c-11e7-9e38-4ac485872ea1.png)

## Notes

- The search box only sort of works; it will only filter whatever page is onscreen at the moment.
- I also noticed some linting issues with dangling underscores; I changed the `.eslintrc` to match our practice and updated the style guide.
- Once we've finished the workflow refactor we'll want to re-do our routing so that the URLs make more sense.

## Testing Instructions

 * Load up the front-end and confirm that the new navigation bar exists and that all the links take you to the expected places.
 * Navigate to the Projects link and verify that the design matches the wireframes.
 * Either delete all your Projects or swap the conditionals in `app-frontend/src/app/pages/library/projects/list/list.html` so that the "No projects" dialogue appears, and confirm that it matches the wireframes and behaves as expected.

Closes #1216, #1214
